src/main/java/com/j2k/worldbox/entity/render/CivilianRender.java

```diff
@@ -0,0 +1,21 @@
+package com.j2k.worldbox.entity.render;
+
+import com.j2k.worldbox.WorldBoxMod;
+import com.j2k.worldbox.entity.custom.CivilianEntity;
+import com.j2k.worldbox.entity.model.CivilianModel;
+import net.minecraft.client.renderer.entity.EntityRendererManager;
+import net.minecraft.client.renderer.entity.MobRenderer;
+import net.minecraft.util.ResourceLocation;
+
+public class CivilianRender extends MobRenderer<CivilianEntity, CivilianModel<CivilianEntity>> {
+    protected static final ResourceLocation TEXTURE =
+            new ResourceLocation(WorldBoxMod.MOD_ID, "textures/entity/civilian.png");
+
```

Comment:
write comments explaining the meanings of magic numbers

Reply:
where are the magic values used here

Reply:
sorry, magic values on line 15 
```java
super(renderManagerIn, new CivilianModel<>(1F), 0.7F);
```